docs/skaffold.md

```diff
@@ -0,0 +1,39 @@
+## Development with skaffold
+
+The network operator includes a `skaffold.yaml` for iterative development with `skaffold`.
+
+For more information on `skaffold` [see the official docs](https://skaffold.dev/).
+
+### Prerequisites
+NOTE: This list does not include dependencies of the `network-operator` e.g. Node Feature Discovery or cert-manager. These must be installed on the cluster independently where required.
+
+- Skaffold CLI - https://skaffold.dev/docs/install/#standalone-binary
+- An image registry to push images to.
+- kustomize
+- kubectl
+- Docker
+- A running Kubernetes cluster.
+
+### Deploying with Skaffold
+
+Export a registry to push images to:
```

Comment:
do you want to add some info on how to create a local registry:

e.g run:

`docker run -p 5000:5000 --name registry registry:2.8.3`

Reply:
Not for now - I think people might have their own solutions for managing dev images already, and this will likely change a lot with a Minikube dev-env